care/facility/tasks/fileupload/delete_incomplete_file_uploads.py

```diff
@@ -0,0 +1,20 @@
+from datetime import timedelta
+
+from celery.decorators import periodic_task
+from celery.schedules import crontab
+from django.utils import timezone
+
+from care.facility.models.file_upload import FileUpload
+
+
+@periodic_task(
+    run_every=crontab(minute="0", hour="0")
+)  # Run the task daily at midnight
+def delete_incomplete_file_uploads():
+    yesterday = timezone.now() - timedelta(days=1)
+    incomplete_uploads = FileUpload.objects.filter(
+        created__date__lte=yesterday, upload_completed=False
+    )
+    for upload in incomplete_uploads:
+        upload.delete_object()
+        upload.delete()
```

Comment:
Why a loop to delete? We can just do `incomplete_uploads.update(deleted=True)` right?